src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -21,6 +21,8 @@
 #include "DataStructures/DataBox/TagTraits.hpp"
 #include "ErrorHandling/Error.hpp"
 #include "ErrorHandling/StaticAssert.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
```

Comment:
why are these being added here?  This is causing the gcc-9, Debug, ON failure...

Reply:
If I recall correctly, I needed to add these so that the databox serialization found the new pup functions for the const global cache pointer and std::atomic. Could you help me understand what's going wrong with the dependencies? Maybe there's an easy workaround, like separating out the const global cache pointer pup to a file other than `ConstGlobalCache.hpp`.